sections/Discussion.tex

```diff
@@ -9,37 +9,46 @@ \section{Model analysis}\label{sec:discussion}
 		\midrule
-		0 & sæby, a, direktør, frederikshavn, virksomheden, hans, medarbejdere, pedersen, firmaet, procent \\
-		1 & omradet, boliger, natur, naturen, ligger, du, vand, dyr, a, skov \\
-		2 & nielsen, arets, prisen, dansk, mors, jensen, løgstør, aars, vm, thy \\
+		1 & sæby, a, direktør, frederikshavn, virksomheden, hans, medarbejdere, pedersen, firmaet, procent \\
+		2 & omradet, boliger, natur, naturen, ligger, du, vand, dyr, a, skov \\
+		3 & nielsen, arets, prisen, dansk, mors, jensen, løgstør, aars, vm, thy \\
 	\end{tabular}
 	\begin{tabular}{c|p{0.8\columnwidth}}
 		\midrule
 		Topic & Top 10 words for author-topic \\
 		\midrule
-		0 & du, procent, unge, børn, arige, hans, dansk, mig, thisted, mener\\
-		1 & sine, skriver, mig, børn, seneste, land, dansk, kommuner, andersen, formand \\
-		2 & set, glas, odense, vesthimmerland, leth, markedet, trump, ni, regionerne, prins\\
+		1 & du, procent, unge, børn, arige, hans, dansk, mig, thisted, mener\\
+		2 & sine, skriver, mig, børn, seneste, land, dansk, kommuner, andersen, formand \\
+		3 & set, glas, odense, vesthimmerland, leth, markedet, trump, ni, regionerne, prins\\
 	\end{tabular}
 	\begin{tabular}{c|p{0.8\columnwidth}}
 		\midrule
 		Topic & Top 10 words for category-topic \\
 		\midrule
-		0 & du, hans, børn, mig, thisted, procent, bedre, a, kr, maske \\
-		1 & klar, haft, fem, hjørring, ham, nyt, formand, min, aften, sagen\\
-		2 & du, min, gode, gamle, ad, henrik, eu, finde, sat, hobro\\
+		1 & du, hans, børn, mig, thisted, procent, bedre, a, kr, maske \\
+		2 & klar, haft, fem, hjørring, ham, nyt, formand, min, aften, sagen\\
+		3 & du, min, gode, gamle, ad, henrik, eu, finde, sat, hobro\\
+	\end{tabular}
+	\begin{tabular}{c|p{0.8\columnwidth}}
+		\midrule
+		Topic & Top 10 words for taxonomy-topic \\
+		\midrule
+		1 & aab, jacob, kasper, rasmus, jakob, pedersen, andersen, friis, minut, allan \\
+		2 & landbrug, landbruget, landmænd, vand, miljø, affald, bedre, natur, vandløb, fødevarer \\
+		3 & virksomheden, millioner, a, direktør, procent, medarbejdere, selskabet, overskud, ansatte, virksomhed \\
 	\end{tabular}
 \end{table}
 
 To compare these models, we take the top 200 words of the topic-word distributions within each model and mark them in the article.
 We take 200 words since we want to see how intertwined the models are.
-Since the Author and Category models do not have a document-topic distribution we can not look at the specific document, but instead, we have marked the words from the given category- and author-topic distribution for the document's category and author, to see what the difference in topics are.
-\begin{table}[ht]
+Since the author-topic and category-topic models do not have a document-topic distribution we can not look at the specific document, but instead, we have marked the words from the given category- and author-topic distribution for the documents category and author, to see what the difference in topics are.
```

Comment:
```suggestion
Since the author-topic and category-topic models do not have a document-topic distribution we can not look at the specific document, but instead, we have marked the words from the given category- and author-topic distribution for the document's category and author to see what the difference in topics are.
```